client/app/telecom/sms/sms/templates/telecom-sms-sms-templates.controller.js

```diff
@@ -34,6 +34,11 @@ angular.module("managerApp").controller("TelecomSmsSmsTemplatesCtrl", class Tele
                 serviceName: this.$stateParams.serviceName
             }).$promise.then((templates) => {
                 this.templates.raw = templates;
+                for (let model of this.templates.raw) {
```

Comment:
We should avoid using this syntax for the moment. Not yet covered by our babel config/target.

Reply:
for .. of is not in preset-es2015?